HEADING-template.md

```diff
@@ -6,6 +6,7 @@
       srcset="assets/revanced-headline/revanced-headline-vertical-dark.svg"
     >
     <img 
+      width="256px"
```

Comment:
This is an arbitrary width, how did you calculate it?

Reply:
Ah, I'm certain that I don't calculated this.

I just copy and paste the width constraint from above.